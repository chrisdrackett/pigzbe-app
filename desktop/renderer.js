import {AppRegistry} from 'react-native';
import App from './bundle.web.js';

AppRegistry.registerComponent('App', () => App);
AppRegistry.runApplication('App', {rootTag: document.getElementById('react-root')});
