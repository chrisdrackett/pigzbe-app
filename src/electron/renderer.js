import {AppRegistry} from 'react-native';
// import App from '../../App';
import App from '../../dist/bundle.web.js';

AppRegistry.registerComponent('App', () => App);
AppRegistry.runApplication('App', {rootTag: document.getElementById('react-root')});
