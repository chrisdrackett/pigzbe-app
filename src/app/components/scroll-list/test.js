import React from 'react';
import renderer from 'react-test-renderer';
import ScrollList from './';

const props = {
    items: []
};

describe('Scroll List', () => {
    test('renders correctly', () => {
        const tree = renderer.create(<ScrollList {...props}/>).toJSON();
        expect(tree).toMatchSnapshot();
    });
});
