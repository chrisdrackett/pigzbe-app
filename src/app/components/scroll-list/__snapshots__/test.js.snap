// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Scroll List renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "stretch",
      "alignSelf": "stretch",
      "backgroundColor": "rgb(247, 247, 250)",
      "flex": 1,
      "justifyContent": "center",
      "position": "relative",
    }
  }
>
  <RCTScrollView
    data={Array []}
    disableVirtualization={false}
    getItem={[Function]}
    getItemCount={[Function]}
    horizontal={false}
    initialNumToRender={10}
    invertStickyHeaders={undefined}
    keyExtractor={[Function]}
    maxToRenderPerBatch={10}
    numColumns={1}
    onContentSizeChange={[Function]}
    onEndReachedThreshold={2}
    onLayout={[Function]}
    onMomentumScrollEnd={[Function]}
    onScroll={[Function]}
    onScrollBeginDrag={[Function]}
    onScrollEndDrag={[Function]}
    renderItem={[Function]}
    scrollEventThrottle={50}
    stickyHeaderIndices={Array []}
    updateCellsBatchingPeriod={50}
    viewabilityConfigCallbackPairs={Array []}
    windowSize={21}
  >
    <View />
  </RCTScrollView>
</View>
`;
